src/spinnaker-dependencies.template

```diff
@@ -24,26 +25,27 @@ versions:
   guava: "{{com.google.guava:guava}}"
   hamcrest: "{{org.hamcrest:hamcrest-core}}"
   httpclient: "{{org.apache.httpcomponents:httpclient}}"
-  #httpcore bom: "{{org.apache.httpcomponents:httpcore}}"
+  httpcore bom: "{{org.apache.httpcomponents:httpcore}}"
   httpcore: "4.3.3"
   jackson: "{{com.fasterxml.jackson.core:jackson-core}}"
   jedis: "{{redis.clients:jedis}}"
   junit: "{{junit:junit}}"
   lombok: "{{org.projectlombok:lombok}}"
   objenesis: "{{org.objenesis:objenesis}}"
-  #okhttp bom: "{{com.squareup.okhttp:okhttp}}"
+  okhttp bom: "{{com.squareup.okhttp:okhttp}}"
   okHttp: "2.7.0"
   redisEmbedded: "0.6"
   retrofit: "1.9.0"
   rxJava: "1.0.16"
   slf4j: "{{org.slf4j:slf4j-api}}"
   snappy: "{{org.xerial.snappy:snappy-java}}"
   spectator: "0.36.0"
-  #spock bom: "{{org.spockframework:spock-core}}"
-  spock: "1.1-groovy-2.3-rc-1"
+  spock bom: "{{org.spockframework:spock-core}}"
+  spock: "{{org.spockframework:spock-core}}"
```

Comment:
I'd leave this explicit instead of inheriting from the bom (`1.1-groovy-2.4-rc-2`)